README.md

```diff
@@ -72,16 +72,4 @@ To debug the application:
 
 ## Troubleshooting
 
-1. Make sure `mbed-cli` is working correctly and its version is `>1.0.0`
-
- ```
- mbed --version
- ```
-
- If not, you can update it:
-
- ```
- pip install mbed-cli --upgrade
- ```
-
-2. If using Keil MDK, make sure you have a license installed. [MDK-Lite](http://www.keil.com/arm/mdk.asp) has a 32 KB restriction on code size.
\ No newline at end of file
+If you have problems, you can review the [documentation](https://os.mbed.com/docs/latest/tutorials/debugging.html) for suggestions on what could be wrong. 
```

Comment:
Query: Should we add "and how to fix it" to the end?

Reply:
Yes. Feel free to tweak the doc and then merge. Thanks!